### ISSUE TITLE
PagerankTable spurious build fails

### ISSUE THREAD

**Issue by decentralion** [sourcecred/sourcecred#719]
We're seeing flaky build failures. Here's a representative snapshot:

```
   FAIL src/app/credExplorer/pagerankTable/Aggregation.test.js (172.527s)
      ● app/credExplorer/pagerankTable/Aggregation › AggregationRow › instantiates a TableRow › with the aggregation score as its cred
    
        Timeout - Async callback was not invoked within the 5000ms timeout specified by jest.setTimeout.
    
          110 |         expect(row.props().showPadding).toBe(false);
          111 |       });
        > 112 |       it("with the aggregation score as its cred", async () => {
              |       ^
          113 |         const {row, aggregation} = await setup();
          114 |         expect(row.props().cred).toBe(aggregation.summary.score);
          115 |       });
    
          at Spec (node_modules/jest-jasmine2/build/jasmine/Spec.js:85:20)
          at Suite.describe (src/app/credExplorer/pagerankTable/Aggregation.test.js:112:7)
          at Suite.describe (src/app/credExplorer/pagerankTable/Aggregation.test.js:99:5)
          at Suite.Object.<anonymous>.describe (src/app/credExplorer/pagerankTable/Aggregation.test.js:67:3)
          at Object.<anonymous> (src/app/credExplorer/pagerankTable/Aggregation.test.js:21:1)
    
    FAIL src/app/credExplorer/pagerankTable/Node.test.js (173.606s)
      ● app/credExplorer/pagerankTable/Node › NodeRowList › creates up to `maxEntriesPerList` `NodeRow`s
    
        Timeout - Async callback was not invoked within the 5000ms timeout specified by jest.setTimeout.
    
          59 |       });
          60 |     });
        > 61 |     it("creates up to `maxEntriesPerList` `NodeRow`s", async () => {
             |     ^
          62 |       const maxEntriesPerList = 1;
          63 |       const {element, nodes, sharedProps} = await setup(maxEntriesPerList);
          64 |       expect(nodes.length).toBeGreaterThan(maxEntriesPerList);
    
          at Spec (node_modules/jest-jasmine2/build/jasmine/Spec.js:85:20)
          at Suite.describe (src/app/credExplorer/pagerankTable/Node.test.js:61:5)
          at Suite.Object.<anonymous>.describe (src/app/credExplorer/pagerankTable/Node.test.js:29:3)
          at Object.<anonymous> (src/app/credExplorer/pagerankTable/Node.test.js:18:1)
    
    PASS src/app/credExplorer/pagerankTable/Connection.test.js (173.13s)
    PASS src/app/createRelativeHistory.test.js (173.828s)
    PASS src/app/withAssets.test.js (140.744s)
    PASS src/plugins/github/graphView.test.js (175.405s)
    PASS src/plugins/git/example/exampleRepo.test.js (129.599s)
    PASS src/app/credExplorer/App.test.js (179.169s)
    PASS src/plugins/git/graphView.test.js (182.956s)
    PASS src/plugins/git/loadRepository.test.js (138.115s)
    
    Summary of all failing tests
    FAIL src/app/credExplorer/pagerankTable/Aggregation.test.js (172.527s)
      ● app/credExplorer/pagerankTable/Aggregation › AggregationRow › instantiates a TableRow › with the aggregation score as its cred
    
        Timeout - Async callback was not invoked within the 5000ms timeout specified by jest.setTimeout.
    
          110 |         expect(row.props().showPadding).toBe(false);
          111 |       });
        > 112 |       it("with the aggregation score as its cred", async () => {
              |       ^
          113 |         const {row, aggregation} = await setup();
          114 |         expect(row.props().cred).toBe(aggregation.summary.score);
          115 |       });
    
          at Spec (node_modules/jest-jasmine2/build/jasmine/Spec.js:85:20)
          at Suite.describe (src/app/credExplorer/pagerankTable/Aggregation.test.js:112:7)
          at Suite.describe (src/app/credExplorer/pagerankTable/Aggregation.test.js:99:5)
          at Suite.Object.<anonymous>.describe (src/app/credExplorer/pagerankTable/Aggregation.test.js:67:3)
          at Object.<anonymous> (src/app/credExplorer/pagerankTable/Aggregation.test.js:21:1)
    
    FAIL src/app/credExplorer/pagerankTable/Node.test.js (173.606s)
      ● app/credExplorer/pagerankTable/Node › NodeRowList › creates up to `maxEntriesPerList` `NodeRow`s
    
        Timeout - Async callback was not invoked within the 5000ms timeout specified by jest.setTimeout.
    
          59 |       });
          60 |     });
        > 61 |     it("creates up to `maxEntriesPerList` `NodeRow`s", async () => {
             |     ^
          62 |       const maxEntriesPerList = 1;
          63 |       const {element, nodes, sharedProps} = await setup(maxEntriesPerList);
          64 |       expect(nodes.length).toBeGreaterThan(maxEntriesPerList);
    
          at Spec (node_modules/jest-jasmine2/build/jasmine/Spec.js:85:20)
          at Suite.describe (src/app/credExplorer/pagerankTable/Node.test.js:61:5)
          at Suite.Object.<anonymous>.describe (src/app/credExplorer/pagerankTable/Node.test.js:29:3)
          at Object.<anonymous> (src/app/credExplorer/pagerankTable/Node.test.js:18:1)
    
    
    Test Suites: 2 failed, 48 passed, 50 total
    Tests:       2 failed, 1182 passed, 1184 total
    Snapshots:   128 passed, 128 total
    Time:        229.718s
    Ran all test suites.
 PASS  check-pretty
 PASS  lint
 PASS  flow
Full results
 PASS  ensure-flow-typing
 PASS  check-stopships
 PASS  check-pretty
 PASS  lint
 PASS  flow
Contents of stdout:
    No errors!
 FAIL  unit
```

**Comment by wchargin** [comment]
This can happen on other tests, usually within the same suite:

```
    FAIL src/app/credExplorer/pagerankTable/Aggregation.test.js (79.08s)
      ● app/credExplorer/pagerankTable/Aggregation › AggregationRow › instantiates a TableRow › with the correct depth
    
        Timeout - Async callback was not invoked within the 5000ms timeout specified by jest.setTimeout.
    
           98 |     }
           99 |     describe("instantiates a TableRow", () => {
        > 100 |       it("with the correct depth", async () => {
              |       ^
          101 |         const {row, depth} = await setup();
          102 |         expect(row.props().depth).toBe(depth);
          103 |       });
    
          at Spec (node_modules/jest-jasmine2/build/jasmine/Spec.js:85:20)
          at Suite.describe (src/app/credExplorer/pagerankTable/Aggregation.test.js:100:7)
          at Suite.describe (src/app/credExplorer/pagerankTable/Aggregation.test.js:99:5)
          at Suite.Object.<anonymous>.describe (src/app/credExplorer/pagerankTable/Aggregation.test.js:67:3)
          at Object.<anonymous> (src/app/credExplorer/pagerankTable/Aggregation.test.js:21:1)
```

**Comment by decentralion** [comment]
This seems to have been fixed by #856.
